example/pom.xml

```diff
@@ -74,7 +74,17 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
       <resource>
         <directory>${project.basedir}/docker</directory>
         <filtering>true</filtering>
-        <targetPath>${project.build.directory}/hono</targetPath>
+        <targetPath>${project.build.directory}/hono/example/docker</targetPath>
```

Comment:
If we change the path here we also need to adapt the *Getting Started* and deployment guides accordingly. I have no problem with that, but now that we are doing this, maybe we can use
`${project.build.directory}/deploy/docker`
`${project.build.directory}/deploy/kubernetes`
`${project.build.directory}/deploy/openshift`?